scanner/src/main/kotlin/Scanner.kt

```diff
@@ -216,17 +209,16 @@ class Scanner(
 
         val issues = controller.getIssues()
 
-        val scannerRun = ScannerRun.EMPTY.copy(
+        return ScannerRun.EMPTY.copy(
+            startTime = startTime,
+            endTime = Instant.now(),
```

Comment:
It's not wrong to get the end time just here, but personally I'd have a slight preference to have

```kotlin
val endTime = Instant.now()

```

in currently line 178, so that the `startTime` and `endTime` variables embrace the block of function calls where the real work is done.